superset-frontend/src/components/Modal/Modal.tsx

```diff
@@ -129,6 +150,42 @@ export const StyledModal = styled(BaseModal)<StyledModalProps>`
   &.no-content-padding .ant-modal-body {
     padding: 0;
   }
+
+  ${({ draggable }) =>
+    draggable &&
+    `
+    .ant-modal-header {
+      .draggable-trigger {
+          cursor: move;
+          width: 100%;
+        }
+    }
+  `};
```

Comment:
It would be nice if the draggable trigger covered the whole header, not just the title:
![image](https://user-images.githubusercontent.com/33317356/130607204-e5c55bf0-e000-4286-949f-091691918e9d.png)

Reply:
I'll check this out